pom.xml

```diff
@@ -114,17 +103,6 @@
               </execution>
             </executions>
           </plugin>
-          <plugin>
-            <artifactId>maven-javadoc-plugin</artifactId>
-            <executions>
-              <execution>
-                <id>attach-javadocs</id>
-                <goals>
-                  <goal>jar</goal>
-                </goals>
-              </execution>
-            </executions>
-          </plugin>
```

Comment:
The parent POM's `jenkins-release` profile includes this logic as of jenkinsci/pom#296. I tested this by running `mvn clean verify -Pjenkins-release` and verified that the Javadoc still got generated.